examples/2016-10-31/function_request_parameters/template.yaml

```
@@ -0,0 +1,33 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: Simple API Endpoint configured using Swagger specified inline and backed by a Lambda function
+Globals:
+  Api:
+    CacheClusterEnabled: true
+    CacheClusterSize: '0.5'
+
+Resources:
+
+  MyLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: index.handler
+      Runtime: nodejs8.10
+      CodeUri: s3://gbdx-build-artifacts/bbeck-test/423f9056eb70d25480852fc472b43dd5 #src/
```

Guessing you meant to revert this back to `src/`

```suggestion
      CodeUri: src/
```

---

Yes, thanks for catching that.  I'm surprised with all the back and forth and testing I haven't committed that already.